packages/authentication/package.json

```diff
@@ -18,7 +18,7 @@
     "@tanstack/react-query": "^4.29.12",
     "js-cookie": "^3.0.5",
     "react": "^18.2.0",
-    "react-hook-form": "^7.34.2",
+    "react-hook-form": "^7.46.1",
```

Comment:
we might need a little story to bump the other packages that use the old versions too + the template

Reply:
Yeah, I was having an issue with the version that we had and updating solved it for me, but we probably want to do that for all of the packages if they start showing that issue as well.